src/components/NotificationRow.tsx

```diff
@@ -59,8 +56,12 @@ export const NotificationRow: FC<IProps> = ({ notification, account }) => {
       // no need to mark as read, github does it by default when opening it
       removeNotificationFromState(settings, notification);
     }
-  }, [notifications, notification, auth, settings]); // notifications required here to prevent weird state issues
```

Comment:
You're sure the state issues won't come back? :)

Reply:
That comment I left "future us" has been very much front of mind...

I think adding `notifications` was a hack because the true dependencies weren't being specified.

Been running this branch locally and so far it's been behaving as expected (touch wood)